tests/flow_test.go

```diff
@@ -2065,3 +2065,63 @@ func TestOvsMirror(t *testing.T) {
 
 	RunTest(t, test)
 }
+
+func TestSFlowCapture(t *testing.T) {
+	test := &Test{
+		setupCmds: []helper.Cmd{
+			{"ovs-vsctl add-br br-sfct", true},
+
+			{"ovs-vsctl add-port br-sfct sfct-intf1 -- set interface sfct-intf1 type=internal", true},
+			{"ip netns add sfct-vm1", true},
+			{"ip link set sfct-intf1 netns sfct-vm1", true},
+			{"ip netns exec sfct-vm1 ip address add 169.254.29.11/24 dev sfct-intf1", true},
+			{"ip netns exec sfct-vm1 ip link set sfct-intf1 up", true},
+
+			{"ovs-vsctl add-port br-sfct sfct-intf2 -- set interface sfct-intf2 type=internal", true},
+			{"ip netns add sfct-vm2", true},
+			{"ip link set sfct-intf2 netns sfct-vm2", true},
+			{"ip netns exec sfct-vm2 ip address add 169.254.29.12/24 dev sfct-intf2", true},
+			{"ip netns exec sfct-vm2 ip link set sfct-intf2 up", true},
+
+			{"ovs-vsctl --id=@sflow create sflow agent=lo target=\"127.0.0.1:6343\" header=128 sampling=1 polling=0 -- set bridge br-sfct sflow=@sflow", true},
+		},
+
+		setupFunction: func(c *TestContext) error {
+			helper.ExecCmds(t, helper.Cmd{Cmd: "ip netns exec sfct-vm1 ping -c 1 -s 1024 169.254.29.12", Check: false})
+			return nil
+		},
+
+		tearDownCmds: []helper.Cmd{
+			{"ip netns del sfct-vm1", true},
+			{"ip netns del sfct-vm2", true},
+			{"ovs-vsctl del-br br-sfct", true},
+		},
+
+		captures: []TestCapture{
+			{gremlin: `G.V().Has('Name', 'lo', 'Type', 'device')`, kind: "sflow", port: 6343},
```

Comment:
With this capture definition you select all the "lo" even those in the namespaces. Something like this would be better

G.V().Has("Type", "host").Out().Has("Name", "lo", "Type", "device")

Reply:
I have one more doubt in this, capture creating by elasticsearch and orientdb is conflicting to each other or it will be deleted once the test is complete?